datasets/piqa/README.md

```diff
@@ -20,9 +20,58 @@ task_ids:
 - multiple-choice-qa
 paperswithcode_id: piqa
 pretty_name: 'Physical Interaction: Question Answering'
+train-eval-index:
+- config: plain_text
+  task: text-generation
```

Comment:
I don't think we can support PIQA right now in AutoTrain as it is a form of text classification with 3 inputs (goal, sol1, sol2). Maybe we should exclude it in this first pass?